docs: Add references to the args and command explanations (#4693)
kubeshop/testkube: docs/docs/test-types/executor-postman.mdx

You can find it in the Testkube repository: https://github.com/kubeshop/testkube/blob/main/test/postman/testkube-api-server-health.postman_collection.json

#### Exporting Postman collection

If you'd like to use another collection you can create it in Postman, and export it to the `.postman_collection.json` file by right clicking it and selecting Export.

> ## Creating and running Test

Reviewer: ```suggestion
## Creating and Running a Test
```

If you prefer to use the Dashboard, go to Tests and click the `Add a new test` button. Then you need to fill in the test Name, choose the test Type (`postman/collection`), and choose Test Source of your choice.

### a) Creating a Test from File

If you have the collection you want to run saved locally, you can use `File` as the test source. You can then select and upload it directly.

![dashboard create postman test from file](../img/dashboard-postman-create-test-file.png)

### b) Creating a Test with Git file

If you have a collection in the Git repository, you can use the `Git file` source. Then, you need to fill in repository details:

- git repository URI (in this case `https://github.com/kubeshop/testkube.git`)
- branch (`main`)
- path to Postman collection in your repository (in this case `test/postman/testkube-api-server-health.postman_collection.json`).

![dashboard create postman test from git-file](../img/dashboard-postman-create-test-git-file.png)

> In this example, the repository is public, but in the case of private ones you would need to additionally fill in Git credentials.

Reviewer: ```suggestion
In this example, the repository is public; if the repository is private, you would need to additionally fill in Git credentials.
```

🚚 Preparing for test run
📦 Fetching test content from git-file...
✅ Test content fetched to path /tmp/git-checkout4073207673/repo/test/postman/testkube-api-server-health.postman_collection.json

> 🔬 Executing in directory :

Reviewer: ```suggestion
🔬 Executing in directory:
```

<TabItem value="crd" label="Custom Resource">

> The third option to create the Test is to use a Test CRD. If you already have the test created, you can check the definition in Dashboard (`Definition` tab in Test Settings).

Reviewer: ```suggestion
The third option for creating the Test is to use a Test CRD. If you already have the test created, you can check the definition in Dashboard (`Definition` tab in Test Settings).
```
